packages/local-serverless-server/README.md

```diff
@@ -0,0 +1,71 @@
+# @hubspot/local-serverless-server
```

Comment:
I think we should come up with another name for the package. My suggestion would be something that doesn't have `local` in the name because I think it could be used for things like testing a serverless function in a CI situation and for it to forgo the pun. Perhaps we could use something along the lines of `serverless-function-runner` or `serverless-runner`. I think the model of this being about execution not a server is best. Could even use `execute` or `executor`. Naming is hard…

Reply:
I just wanted the name to focus on the fact that we need to run a server to test serverLESS functions 😂 

Sounds good though, I will work on a better name.